common/scripts/.githooks/make_lint-all.sh

```diff
@@ -15,4 +15,5 @@
 # limitations under the License.
 
 # Launches fmt and lint checks
-make lint-all
+#make lint-all
+make lint-scripts
```

Comment:
Why is this changing? I would think that we want to lint everything as a default, and the script itself is called `make_lint-all.sh`.